src/core/css/ErrorHandler.scss

```diff
@@ -7,4 +7,8 @@
   padding: 10px;
   margin: 0;
   margin-bottom: 10px;
+
+  > li {
```

Comment:
I really don't see the point in trying to optimize our css for theoretical performance that we have no measurement of whatsoever. I know there are blog posts but those blog posts get outdated and even the right to left problem isn't so much of a problem anymore (and this was from 2011): https://calendar.perfplanet.com/2011/css-selector-performance-has-changed-for-the-better/

Adding css classes to every sub-element within a component is annoying and I just don't think it's worth it. I would rather see some performance metrics to prove what we should and should not be doing.